lib/src/components/dropdown-menu/DropdownMenuItem.tsx

```diff
@@ -12,10 +13,7 @@ export const itemStyles = {
   position: 'relative',
   px: '$3',
   py: '$2',
-  '&[data-disabled]': {
-    color: '$tonal200',
-    pointerEvents: 'none'
-  },
+  '&[disabled]': { disabledStyle, pointerEvents: 'none' },
```

Comment:
This will need to be:

```suggestion
  '&[disabled]': { ...disabledStyle, pointerEvents: 'none' },
```